plugins/GSdx/Renderers/Common/GSOsdManager.cpp

```diff
@@ -397,12 +408,31 @@ size_t GSOsdManager::GeneratePrimitives(GSVertexPT1* dst, size_t count) {
 		for(const auto &pair : m_monitor) {
 			if((pair.first.size() + pair.second.size()) * 6 > count - drawn) break;
 
-			// Calculate where to start rendering from by taking the right most position 1.0
-			// and subtracting (going left) 8 scaled pixels for a margin, then subtracting
-			// the size of the longest key and subtracting a scaled space and finally
-			// subtracting the longest value
-			float x = 1.0f - 8 * (2.0f/m_real_size.x) - first_max - m_char_info[' '].ax * (2.0f/m_real_size.x) - second_max;
-			float y = -1.0f + ((m_size+2)*(2.0f/m_real_size.y)) * line++;
+			float x, y;
+
+			switch (m_monitor_pos) {
+				case gsosd_pos::TOPLEFT:
```

Comment:
It doesn't overlap with the other logs?

Reply:
Nope. Whenever the monitor is enabled and its position is at the top left, the logs get moved to the top right.
Line 353 is where the log changes begin.